Hủaıbaı/assets/minecraft/lang/toa_tqg.json

```diff
@@ -118,8 +118,32 @@
   "disconnect.genericReason": "%s",
   "disconnect.quitting": "Chủfaq shảı lủaq",
   "options.language": "Zủ...",
+  "options.sounds": "Gỉaq ru Lảq...",
+  "options.sounds.title": "Pỏ Gỉaq ru Lảq ga Kẻajuao",
   "record.nowPlaying": "Nỉ gỉaq ku: %s",
   "options.languageWarning": "Dảı bû mủo dủe sé tủa nỏ zủ zay",
+  "options.online": "Zỉqrıaq...",
+  "options.online.title": "Pỏ Zỉqrıaq ga Kẻajuao",
+  "options.difficulty": "Nỏacaı",
+  "options.difficulty.online": "Pỏ Lủeqce ga Nỏacaı",
+  "options.difficulty.peaceful": "Sỏıcıaruo",
+  "options.difficulty.easy": "Fủı",
+  "options.difficulty.normal": "Nẻfaı",
+  "options.difficulty.hard": "Nỏa",
+  "options.difficulty.hardcore": "Pỏ Hỉaodaı ga Sỏqne",
+  "options.fov": "Kảqjıeqgıu",
+  "options.fov.min": "Nẻfaı",
+  "options.fov.max": "Kảqsoqne",
+  "soundCategory.master": "Jỏqlaqcaı",
```

Comment:
We could avoid creating a special word for this purpose with a serial verb like 'Jóq nẻ lảqcaı'